.circleci/config.yml

```diff
@@ -66,6 +66,8 @@ commands:
                 command: |
                   git --git-dir=dkan/.git --work-tree=dkan checkout "${CIRCLE_SHA1}"
                   dktl exec:composer -- require --no-interaction --ansi --no-update "getdkan/dkan:dev-${CIRCLE_SHA1}"
+                  rm -rf docroot/modules/contrib/select_or_other
```

Comment:
The fact that select_or_other was patched led to an error during the update tests. To fix this I had to have the script remove the module and then also remove the vendor directory.